templates/tools/dockerfile/test/sanity/Dockerfile.template

```diff
@@ -19,7 +19,7 @@
 
   #========================
   # Sanity test dependencies
-  RUN apt-get update && apt-get -t buster install -y python3.7 python3-all-dev
+  RUN apt-get update && apt-get -t buster install -y python3.7 python3-all-dev gawk
```

Comment:
I don't think we should add a new dependency into the dockerimage just because of this pretty specialized PR.
If we can't make it happen with what's already in the sanity dockerimage, I'd say let's not do this at all.